### ISSUE TITLE
Success and error messages are not escaped for use as in html attributes

### ISSUE THREAD

**Issue by croxton** [solspace/craft3-freeform#904]
**Describe the bug or issue you're experiencing**
If you enter a string containing html markup or just a double quote for the success or error messages, it is output unescaped in the template.

**Steps to reproduce**

1. Create a form, go to the validation tab and enter a Success Message such as `This is <a href="#">unescaped</a>`
2. Render the form in a template. Outputs as: 
```html 
<form  method="post" data-success-message="This is <a href="#">unescaped</a>" ... >
```

**Expected behavior**

* Success message should be run through `htmlspecialchars()` or similar so it doesn't break the form markup. 
* It should be possible to use html markup or markdown in the success and error messages (for example, to link to a file download on success).

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.5.5
- Freeform Version: 4.1.8
- Freeform Edition: Pro

**Comment by kjmartens** [comment]
Thanks for reporting this @croxton. I have queued this up to be addressed shortly. 🙂

**Comment by kjmartens** [comment]
This is fixed in Freeform 4.1.12+ 🙂
